supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActorSpec.scala

```diff
@@ -215,29 +216,29 @@ class JesAsyncBackendJobExecutionActorSpec extends TestKitSuite("JesAsyncBackend
     val expectations = Table(
       ("previous_preemptions", "previous_unexpectedRetries", "preemptible", "errorCode", "innerErrorCode", "shouldRunAsPreemptible", "shouldRetry"),
       // No preemptible attempts allowed, but standard failures should be retried.
-      (0, 0, 0, 10, 13, false, true), // This is the new "unexpected failure" mode, which is now retried
-      (0, 1, 0, 10, 13, false, true),
-      (0, 2, 0, 10, 13, false, false), // The third unexpected failure is a real failure.
-      (0, 0, 0, 10, 14, false, false), // Usually means "preempted', but this wasn't a preemptible VM, so this should just be a failure.
-      (0, 0, 0, 10, 15, false, false),
-      (0, 0, 0, 11, 13, false, false),
-      (0, 0, 0, 11, 14, false, false),
+      (0, 0, 0, Status.ABORTED, 13, false, true), // This is the new "unexpected failure" mode, which is now retried
```

Comment:
The "OUT_OF_RANGE" isn't mapped cleanly but that's because there wasn't actually an `11` in the official RPC responses. I was interpreting what I saw to be "this isn't `10`"